src/main/kotlin/vm/VM.kt

```diff
@@ -176,51 +193,54 @@ class VM(val exe: Executable) {
                     val argCount = next().intFromV
                     val a2 = if (word.opcode == O_CALL) pop() else next().value
                     val a1 = pop()
-                    val args = mutableListOf<Value>()
-                    repeat(argCount) { args.add(0, pop()) }
+                    val args = buildList { repeat(argCount) { add(0, pop()) } }
                     if (a2 is VString) {
-                        c.ticksLeft = ticksLeft
-                        c.callsLeft--
-                        a1.callVerb(c, a2.v, args)?.also {
-                            if (word.opcode != O_VCALLST) push(it)
-                        } ?: fail(E_VERBNF, "verb not found")
-                        c.callsLeft++
-                        ticksLeft = c.ticksLeft
+                        // If static built-in verb, call directly without returning
+                        val vReturn = a1.callStaticVerb(c, a2.v, args)
+                        if (vReturn != null) {
+                            if (word.opcode != O_VCALLST) push(vReturn)
```

Comment:
I confess that I don't really understand why you have to do this.  I thought you built-ins were really just calls on the $sys trait.  Maybe you can explain this to me when we get together.